### PR TITLE
PYTHON-1631 Automate release wheels for Windows and manylinux

### PR DESCRIPTION
**Pull request by ShaneHarvey**

### CODE REVIEW

File: .evergreen/config.yml [hunk 1]
```diff
@@ -2519,6 +2555,13 @@ buildvariants:
     - name: "aws-auth-test-4.4"
     - name: "aws-auth-test-latest"
 
+- matrix_name: "Release"
+  matrix_spec:
+    platform: [ubuntu-16.04, windows-64-vsMulti-small]
```
Comment (behackett) [inline .evergreen/config.yml:2560]: Why not Ubuntu 18? 16 goes EOL in April.

File: .evergreen/build-mac.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -ex
```
Comment (behackett) [inline .evergreen/build-mac.sh:1]: Why are you adding this now since we can't actually do it?

File: .evergreen/build-manylinux.sh [hunk 3]
```diff
@@ -0,0 +1,40 @@
+#!/bin/bash -ex
+
+docker version
+
+# 2020-03-20-2fda31c Was the last release to include Python 3.4.
+images=(quay.io/pypa/manylinux1_x86_64:2020-03-20-2fda31c \
```
Comment (behackett) [inline .evergreen/build-manylinux.sh:6]: Do all all versions other than 3.4 get built twice?

File: .evergreen/build-mac.sh (hunk 2 above)
Comment (ShaneHarvey) [reply .evergreen/build-mac.sh:1]: We can still use it to build macos wheels manually.

File: .evergreen/build-manylinux.sh (hunk 3 above)
Comment (ShaneHarvey) [reply .evergreen/build-manylinux.sh:6]: Yes. They get built first with these old images and then rebuilt and overwritten with the latest manylinux1_x86_64/manylinux1_i686 images.

File: .evergreen/config.yml (hunk 1 above)
Comment (ShaneHarvey) [reply .evergreen/config.yml]: Changed to 18. They both have Docker so either one works.

**Pull request merged**